exercises/concept/freelancer-rates/.docs/instructions.md

```diff
@@ -22,24 +16,28 @@ dayRate(89);
 
 The day rate does not need to be rounded or changed to a "fixed" precision.
 
-## 2. Calculate the month rate, given an hourly rate and a discount
+## 2. Calculate the number of workdays given a fixed budget and hourly rate
```

Comment:
The task title does not need to contain all the parameters of the function. Those are described below. The same applies to task 3.
```suggestion
## 2. Calculate the number of workdays given a fixed budget
```